tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py

```diff
@@ -1,42 +1,445 @@
+            log.error(f"Can not delete the test pod : {ex}")
+
+        # Deleting the PVC which used in the test.
+        log.info(f"Delete the PVC : {self.pvc_obj.name}")
+        try:
+            self.pvc_obj.delete()
+        except Exception as ex:
+            log.error(f"Can not delete the test pvc : {ex}")
+
+        # Deleting the StorageClass used in the test
+        log.info(f"Deleting the test StorageClass : {self.sc_obj.name}")
+        self.sc_obj.delete()
+
+        # Deleting the VolumeSnapshotClass used in the test
+        log.info(f"Deleting the test Snapshot Class : {self.sc_obj.name}")
+        self.snap_class.delete()
+
+        # Deleting the Data pool
+        self.delete_ceph_pool(self.sc_name)
+
+        # Deleteing the namespace used by the test
+        log.info(f"Deleting the test namespace : {self.nss_name}")
+        switch_to_default_rook_cluster_project()
+        try:
+            self.proj.delete(resource_name=self.nss_name)
+        except CommandFailed:
+            log.error(f"Can not delete project {self.nss_name}")
+            raise CommandFailed(f"{self.nss_name} was not created")
+        self.proj.wait_for_delete(resource_name=self.nss_name, timeout=60, sleep=10)
+
+        super(TestPvcMultiSnapshotPerformance, self).teardown()
+
+    def get_csi_pod(self, namespace):
+        """
+        Getting pod list in specific namespace, for the provision logs
+
+        Args:
+            namespace (str): the namespace where the pod is deployed.
+
+        Returns:
+            list : list of lines from the output of the command.
+
+        """
+        results = run_oc_command(cmd="get pod", namespace=namespace)
+        if ERRMSG in results:
+            err_msg = "Can not get the CSI controller pod"
+            log.error(err_msg)
+            raise Exception(err_msg)
+        return results
+
+    def get_log_names(self):
+        """
+        Finding the name of snapshot logging file
+        the start time is in the 'csi-snapshot-controller' pod, and
+        the end time is in the provisioner pod (csi-snapshotter container)
+
+        """
+        self.log_names = {"start": [], "end": []}
+        log.info("Looking for logs pod name")
+
+        # Getting csi log name for snapshot start creation messages
+        results = self.get_csi_pod(namespace="openshift-cluster-storage-operator")
+        for line in results:
+            if "csi-snapshot-controller" in line and "operator" not in line:
+                self.log_names["start"].append(line.split()[0])
+
+        # Getting csi log name for snapshot end creation messages
+        results = self.get_csi_pod(namespace="openshift-storage")
+        for line in results:
+            if "prov" in line and self.fs_type in line:
+                self.log_names["end"].append(line.split()[0])
+
+        log.info(
+            f"The CSI logs for the test are : {json.dumps(self.log_names, indent=4)}"
+        )
+
+    def build_fio_command(self):
```

Comment:
same here, good to move it to common library

Reply:
same as above.